Classes/Service/AnonymizationService.php

```diff
@@ -107,7 +106,7 @@ protected function anonymizeObject(AbstractDomainObject $object, $config)
                         $this->anonymizeObject($child, $childConfig);
                     }
                 } else {
-                    throw new IllegalObjectTypeException('Only ObjectStorages are supported for anonymization', 1542985424);
+                    throw new IllegalObjectTypeException('Only ObjectStorages, DomainObjects and FileReferences are supported for anonymization.', 1542985424);
```

Comment:
This looks odd and will fail on objects of type `DomainObject` and `FileReference` just from reading.

Reply:
You're right, changed. This part is only for ObjectStorages.